deployer/aws-lambda/kevel/index.js

```diff
@@ -75,11 +75,22 @@ export async function handler(event) {
     const decisionRes = await client.decisions.get(decisionReq, {
       ip: anonymousIp,
     });
-    const { decisions: { div0 } = {} } = decisionRes;
+    const {
+      decisions: { div0 } = {},
+      candidateRetrieval: { div0: { candidatesFoundCount } = {} } = {},
+    } = decisionRes;
     if (div0 === null || div0?.[0] === null) {
+      let status = !candidatesFoundCount ? "unsupported_geo" : "cap_reached";
```

Comment:
nit: I'd also reverse this so we're not dealing with a double negative:

```suggestion
      let status = candidatesFoundCount ? "cap_reached" : "geo_unsupported";
```